sdk/eventgrid/Azure.Messaging.EventGrid/tests/EventGridClientUnitTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
The pattern we use in EH/SB is to call the Live test classes something like EventGridClientLiveTests, and the ones with unit tests EventGridClientTests. Would it make sense to do that here?